requirements/requirements-py-3.7.txt

```diff
@@ -15,6 +17,7 @@ Babel==2.9.1
 backcall==0.2.0
 bcrypt==3.2.0
 beautifulsoup4==4.9.3
+black==20.8b1
```

Comment:
I guess this comes from `disk-objectstore`. Would be good to move that requirement in an extra there so it is not necessary for a default install

Reply:
No this comes from one of the test dependencies (can't remember which one now), it just wasn't explicitly in the requirements (I guess it was added in a recent version)
You can see all the dependencies of the disk-objectstore here: https://github.com/aiidateam/disk-objectstore/blob/develop/requirements.lock

Reply:
OK, thanks. By the way, just saw that [the `setup.cfg`](https://github.com/aiidateam/disk-objectstore/blob/develop/setup.cfg) still mentions support for Python 3.6 although the `python_requires` says Python 3.7 and up

Reply:
Ah cheers, I think perhaps https://github.com/asottile/setup-cfg-fmt does not like tildes i.e. `~=3.7`, because normally it re-syncs the classifiers with the `python_requires`

Reply:
p.s. https://github.com/aiidateam/disk-objectstore/blob/develop/.pre-commit-config.yaml, is basically got all my current favourite hooks lol; things like https://github.com/asottile/pyupgrade are pretty nice